var/spack/repos/builtin/packages/exago/package.py

```diff
@@ -157,9 +157,10 @@ class Exago(CMakePackage, CudaPackage, ROCmPackage):
     # This is no longer a requirement in RAJA > 0.14
     depends_on("umpire+cuda~shared", when="+raja+cuda ^raja@:0.14")
 
-    depends_on("petsc@3.13:3.14", when="@:1.2.99")
-    depends_on("petsc@3.16.0:3.16", when="@1.3.0:1.4")
-    depends_on("petsc@3.18.0:3.19", when="@1.5.0:")
+    depends_on("petsc@3.13:3.14", when="@:1.2")
+    depends_on("petsc@3.16", when="@1.3:1.4")
+    depends_on("petsc@3.18:3.19", when="@1.5")
+    depends_on("petsc@3.20:", when="@1.6:")
```

Comment:
I am fine with this as it doesn't break anything and is logical, but I am pretty sure that `depends_on("petsc@3.18.0:", when="@1.5.0:")` is correct as exago should work with all of those versions... Regardless this is probably just the sane thing to do